### ISSUE TITLE
Stopwatch skill enabled but not available

### ISSUE THREAD

**Issue by UNIVAC-Colonel-Panic** [blandflakes/echo-stopwatch#1]
Hi,

I hope this is the same stopwatch skill that I enabled on my Amazon Dot. When I first tried to use it several weeks ago -- "Alexa, ask stopwatch" -- it says, "The skill stopwatch is enabled on your Alexa app but it's not available right now." It has been giving the same response ever since. I called Amazon Echo support and they said they are unable to find out when it will be available again. So, I thought I would try creating an issue here. Is there something that I need to do on my end to get this to work?

**Comment by blandflakes** [comment]
Hi - this is the same skill. It's odd that it says it's not available. I just used the skill on both my Echo and my Dot.

If there were an error on my end, the response from Alexa is usually something like "There was a problem with the skill's response."

The skill _was_ having some issues for a bit because OpenShift kept idling the app, but I think I've since resolved them. Is it happening even now?
